kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/topics/TopicTests.java

```diff
@@ -85,4 +106,20 @@ void deleteTopic() {
                 .isOnTopicListPage()
                 .isNotVisible(TOPIC_TO_DELETE);
     }
+
+    @SneakyThrows
+    @DisplayName("produce message")
+    @Test
+    void produceMessage(){
+        pages.openTopicsList(SECOND_LOCAL)
+                .isOnPage()
+                .openTopic(TOPIC_TO_UPDATE);
+        Selenide.refresh();
```

Comment:
is there a reason to refresh the page here?

Reply:
 waiting until page is loaded .